be/src/http/action/stream_load.cpp

```diff
@@ -600,7 +600,15 @@ Status StreamLoadAction::_process_put(HttpRequest* http_req, StreamLoadContext*
     } else if (!http_req->header(HTTP_COMPRESSION).empty()) {
         request.__set_payload_compression_type(http_req->header(HTTP_COMPRESSION));
     }
-
+    if (!http_req->header(HTTP_INSERT_MODE).empty()) {
```

Comment:
if there is no HTTP_INSERT_MODE header,  invoke empty will rasie npe or not ?

Reply:
Same with other header